src/target/riscv/riscv.c

```diff
@@ -5353,6 +5371,16 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.usage = "[index ('on'|'off')]",
 		.help = "Controls which RISC-V triggers shall not be touched by OpenOCD.",
 	},
+	{
+		.name = "virt2phys_mode",
+		.handler = handle_riscv_virt2phys_mode,
+		.mode = COMMAND_ANY,
+		.usage = "['sw'|'hw'|'off']",
+		.help = "Configure the virtual address translation mode: "
+				"sw - translate vaddr to paddr by manually traversing page tables, "
+				"hw - translate vaddr to paddr by hardware,"
```

Comment:
Really a detail: Added trailing space character:

```suggestion
				"hw - translate vaddr to paddr by hardware, "
```

Reply:
Addressed